src/main/java/uk/gov/hmcts/reform/civil/utils/HmcDataUtils.java

```diff
@@ -60,4 +64,48 @@ public static boolean hearingDataChanged(PartiesNotifiedResponse partiesNotified
             || partiesNotified.getServiceData() == null
             || hearingDataChanged(hearing, partiesNotified.getServiceData());
     }
+
+    private static int actualHours(int hearingDuration) {
+
+        return hearingDuration == 6 ? 5 : hearingDuration;
+    }
+
+    private static String formatDay(HearingDaySchedule day) {
+        var dateString = day.getHearingStartDateTime().toLocalDate().format(DateTimeFormatter.ofPattern("dd MMMM yyyy"));
+        var timeString = day.getHearingStartDateTime().toLocalTime().toString();
+        var duration = actualHours(getHearingDayHoursDuration(day));
+
+        return String.format("%s at %s for %d %s", dateString, timeString, duration, duration > 1 ? "hours" : "hour");
+    }
+
+    private static int getHearingDayHoursDuration(HearingDaySchedule day) {
+        return ((Long)day.getHearingStartDateTime().until(day.getHearingEndDateTime(), ChronoUnit.HOURS)).intValue();
+    }
+
+    public static List<String> getHearingDaysTextList(HearingGetResponse hearing) {
+        return hearing.getHearingResponse().getHearingDaySchedule().stream()
+            .map(day -> formatDay(day))
+            .collect(Collectors.toList());
+    }
+
+    public static String getTotalHearingDurationText(HearingGetResponse hearing) {
+        var duration = hearing.getHearingResponse().getHearingDaySchedule().stream()
+            .map(day -> getHearingDayHoursDuration(day))
+            .reduce((aac, day) -> aac + day).get();
+
+        if (duration == 6) {
```

Comment:
Suggest to make 6 a constant as well and call it something clear maybe like MAX_HOURS_A_DAY. This would make it easier to maintain in the future too if the hours change.